src/components/ArticleComments.jsx

```diff
@@ -12,10 +13,14 @@ export default function ArticleComments({ article_id }) {
       setIsLoading(false);
     });
   }, [article_id]);
+  const addComment = (comment) => {
+    setComments((comments) => [comment, ...comments]);
```

Comment:
Good to see the comment being added to the top of the comments list as part of optimistic rendering, this will give the user a visual confirmation that their comment has been posted